src/content/cloud/volume-snapshots.mdx

```diff
@@ -0,0 +1,178 @@
+- `csi-okteto-ssd`: backed by solid-state drives (SDD)
+
+</TabItem>
+</Tabs>
+
+## Using Volume Snapshots in your Development Environment
+
+Okteto enables developers to initialize persistent volume claims with the contents of a pre-existing volume snapshot.
+The volume snapshot is created from a persistent volume claim and it can contain database backups, big files, images, a copy of your staging data, etc.
+
+The steps to clone the data of an existing persistent volume into your develepment environment are:
+
+- [Create the source persistent volume with the right storage class](/docs/cloud/use-volume-snapshots/#creating-the-persistent-volume)
+- [Create a volume snapshot of the source persistent volume](/docs/cloud/use-volume-snapshots/#creating-a-volume-snapshot)
+- [Consume the volume snapshot in a new persistent volume](/docs/cloud/use-volume-snapshots/#consuming-a-volume-snapshot)
+
+### Creating the Persistent Volume
+
+If the default storage class of your cluster doesn't support volume snapshots, make sure to pick the right storage class when creating your persistent volume.
+Check the [requirements](/docs/cloud/use-volume-snapshots/#requirements) section to know more about available storage classes in Okteto.
+
+This is a sample using Kubernetes manifests or Docker Compose files:
+
+<Tabs
+  defaultValue="kubernetes"
+  values={[
+    { label: 'Kubernetes', value: 'kubernetes', },
+    { label: 'Compose', value: 'compose', },
+  ]}
+>
+<TabItem value="kubernetes">
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pvc
+spec:
+  storageClassName: csi-okteto-standard
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+```
+
+</TabItem>
+<TabItem value="compose">
+
+```yaml
+services:
+  mysql:
+    image: mysql
+    volumes:
+      - mysql-pvc:/var/lib/mysql
+
+volumes:
+  mysql-pvc:
+    driver_opts:
+      class: csi-okteto-standard
+```
+
+</TabItem>
+</Tabs>
+
+
+### Creating a Volume Snapshot
+
+Volume snapshots are created with the content of the persistent volume at time of creating the volume snapshot. Further updates on the persistent volume aren't reflected in the volume snapshot content.
+
+This step is independent of whether you created your persistent volume with Kubernetes manifests or a Docker Compose file:
+
+The following manifest creates a volume snapshot from the `mysql-pvc` persistent volume claim:
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: mysql-snapshot
+spec:
+  volumeSnapshotClassName: okteto-snapshot-class
+  source:
+    persistentVolumeClaimName: mysql-pvc
+```
+
+### Consuming a Volume Snapshot
+
+Use the `dev.okteto.com/from-snapshot-name` and `dev.okteto.com/from-snapshot-namespace` annotations on any persistent volume claim to tell Okteto to initialize your persistent volume claim from an existing volume snapshot, as shown below:
```

Comment:
We need to add a paragraph explaining what the result of this is, so people know why they are running this commands.